value.go

```diff
@@ -793,6 +793,9 @@ func (v Value) Clone() Value {
 }
 
 func makeInt96(bits []byte) (i96 deprecated.Int96) {
+	if len(bits) < 12 {
+		bits = append(bits, make([]byte, 12-len(bits))...)
```

Comment:
`len(bits) < 12` doesn't imply that `len(bits) == cap(bits)`, we could be overwriting data in the backing array with this `append`.

I would suggest we change the code to the following:
```suggestion
		var buf [12]byte
		bits = append(buf[:0], bits...)
```

Reply:
given the current implementation the makeInt96 functions only gets called with the bits array created by unsafe.Bytes, which makes a slice with equal len and cap, so there is no way to overwrite the underlying array, but I agree that we should change this so that in case there were other use cases in the future, we don't overwrite the data, I will just check for the cap so that we don't allocate memory if unnecessary.